benchmark/ao-render.rb

```diff
@@ -5,8 +5,8 @@
 # mruby version by Hideki Miura
 #
 
-IMAGE_WIDTH = 256
-IMAGE_HEIGHT = 256
+IMAGE_WIDTH = 128
+IMAGE_HEIGHT = 128
```

Comment:
Why do you change IMAGE_WIDTH and IMAGE_HEIGHT?  This PR conflict because of this change.